lib/middleware/opentelemetry_tesla_middleware.ex

```diff
@@ -0,0 +1,9 @@
+defmodule Tesla.Middleware.OpenTelemetry do
```

Comment:
I wonder if this should be something like OpentelemetryTesla.Middleware to indicate it's from this package. Not sure about the convention here.

Reply:
As far as I can see [here](https://github.com/teamon/tesla#custom-middleware), it's ok.
I've looked for a few middlewares online and the naming strategy is not always the same, some use the one like this PR and other ones are completely custom "à lá carte".